packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -43,8 +43,8 @@ type TaskItemProps = {
 	onCollapse?: () => void;
 	onDelete?: () => void;
 	onDismiss?: () => void;
+	onSnooze?: () => void;
```

Comment:
Let's add a changelog entry for this under `packages/experimental`.

Reply:
Done 👍🏻